subprojects/docs/src/docs/userguide/working_with_files.adoc

```diff
@@ -336,12 +336,9 @@ Using a task directly as an argument like this relies on it having <<more_about_
 [[sec:single_file_paths]]
 === Single file paths
 
-One of the great quandaries when developing a build is how to specify file locations when the build may be executed from an arbitrary directory — not necessarily in the project — and may be run on any number of different systems with incompatible directory layouts. The standard Java mechanism for specifying a file path runs into trouble in these situations:
+Never use `new File(relative path)`, as this creates a path relative to the current working directory, which could be anywhere.
```

Comment:
This would be fine if there were some context, i.e. paths to single files had been discussed first. Also, we shouldn't assume we're talking to someone familiar with the Java API.